src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -230,10 +230,14 @@ private void CommitChanges(object param)
             //if paths are modified, reload packages and update prefs.
             if (!setting.CustomPackageFolders.SequenceEqual(newpaths))
             {
+                if(newpaths.Count > setting.CustomPackageFolders.Count)
```

Comment:
what if a user deletes an existing path and then adds a different one won't this miss the new path?